docs/how-to-guides/session-customizing.rst

```diff
@@ -31,57 +31,83 @@ done automatically for Python and R projects if you add the packages you want
 to ``requirements.txt`` and ``install.R`` respectively.
 
 
+.. _renku_project_config:
+
 Renku project configurations
 ----------------------------
 
 When starting a new Session, most of the options can be manually
 changed by the user. Depending on the specific RenkuLab deployment, you can select
-more RAM, a higher CPU quota, etc.
+more RAM, more storage space, a higher CPU quota, etc.
 
 Your project may even include a package with an advanced UI (like
 `Streamlit <https://renku.discourse.group/t/how-to-deploy-streamlit-in-renku/169>`_)
 and you probably want to choose it as default.
 
-It's possible to set a default value for all these options using the project
-configurations stored in the ``.renku/renku.ini`` file.
-Once you do that, each time a user tries to start a new environment, those options will
-be pre-selected.
+It's possible to set a default value for all these options, either using the
+`renku config command`_ locally or in a session, or directly on RenkuLab project
+page.
+
+To do that, you can go into the project `Settings` section and click on the
+`Sessions` tab. The interface shows the available project settings. Mind that you
+can only select values valid to the specific RenkuLab platform. You can still use
+the CLI to set any custom value, but that may not work as expected.
+
+.. image:: ../../_static/images/project-session-settings.png
+  :width: 85%
+  :align: center
+  :alt: Project session settings
+
+If no default value is defined for a resource at the project level, the session
+will likely use the default for the RenkuLab platform. Picking a specific value
+is helpful if you know the project requires more resources than the standard.
+Mind that users can always manually select different values when starting a
+session if they want to, even if you set a default.
+
+These configurations are stored in the  ``.renku/renku.ini`` file, so they are
+preserved even if you move the project to another RenkuLab instance.
 
 .. note::
 
-  Manually modifying the ``renku.ini`` file is not recommended.
-  You can use the
-  `renku config command <https://renku-python.readthedocs.io/en/latest/commands.html#module-renku.cli.config>`_
-  from a session, for example:
+  Sometimes you may want to add a non-default value, typically to select a custom
+  default environment. Manually modifying the ``renku.ini`` file is *not*
+  recommended. You can use the `renku config command`_ from a session, for example:
 
     renku config set interactive.default_url "/tree"
```

Comment:
This should be formatted as a code block 

![image](https://user-images.githubusercontent.com/3353586/167865314-85bb2101-3dda-4cf3-9619-1e83eab7d140.png)